app/utils/constants.js

```diff
@@ -186,6 +186,9 @@ var C = {
     LAUNCH_CONFIG: 'io.rancher.service.launch.config',
     LAUNCH_CONFIG_PRIMARY: 'io.rancher.service.primary.launch.config',
     SIDEKICK: 'io.rancher.sidekicks',
+    SERVICE_APP_UI: 'io.rancher.service.app.ui',
```

Comment:
Group related labels with a common prefix, I.e. `io.rancher.service.ui_link.{label, port, path}`.

Reply:
You may want to support the label being a JSON map of language code to label, or else you won't be able to have an app label localized for English vs China vs other..

Reply:
Shall we make the `io.rancher.service.ui_link.label` optional between JSON map and string, or it's must be a JSON map?